docs/local-dev-everything-local.md

```diff
@@ -35,33 +35,37 @@ The steps below assumes that the IDE used is [IntelliJ IDEA](https://www.jetbrai
-    | InventoryApplication  | ITEMS             | items:\n  - name: "BigBurger"\n    type: "burgers"\n    price: 5.50\n    imageUrl: "usr/lib/images/bigburger.png"\n    quantity: 200\n    labels: [ "retail", "restaurant", "food" ]\n  - name: "DoubleBurger"\n    type: "burgers"\n    price: 7.20\n    imageUrl: "usr/lib/images/burgers.png"\n    quantity: 200\n    labels: [ "retail", "restaurant", "food" ]\n  - name: "Shirt"\n    type: "textile"\n    price: 15.50\n    imageUrl: "usr/lib/images/shirt.png"\n    quantity: 50\n    labels: [ "retail", "textile", "clothing" ]\n  - name: "Short"\n    type: "textile"\n    price: 17.20\n    imageUrl: "usr/lib/images/short.png"\n    quantity: 20\n    labels: [ "retail", "textile", "clothing" ]                  |
+  - **Set environment variables for the services**
+
+      To be able to run the services locally we must update the `Run configurations`
+      with some additional parameters. These values are passed in to the services as
+      `Config Map Values` in the K8s cluster based setup.
+
+    - Click on `Edit Configurations` _(see image above)_ from the top right menu
+    - In the window that pops up, set the following `Environment Variables` for
+        the three different services. If you don't see an option to fill in
+        environment variables, you can enable it from the `Modify options` dropdown.
+
+        <p align="center">
+            <img src="images/ide2.png">
+            <div align="center">
+                <strong>(click to enlarge)</strong>
+            </div>
+        </p>
+
+      | Service               | Variable Key      | Variable Value                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+      |---------              |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------           |
```

Comment:
As far as I know, you are able to create markdown tables which can preserve spacing and code-like formatting. You will have to use HTML table elements, though. Here is an example:

https://stackoverflow.com/questions/28508141/code-block-inside-table-row-in-markdown

Reply:
Thank you. Changed this to use <pre> tags